### PR TITLE
SOC-5554 Unable to invite a user in a space when site is displayed in FR

### PR DESCRIPTION
**Pull request by boubaker**
add double quotes instead of simple quotes

### CODE REVIEW

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl [hunk 1]
```diff
@@ -34,6 +34,6 @@ String placeholder = _ctx.appRes("UIUserInvitation.input.placeholder");
   <% uiform.end();%>
   <%
    requireJs.require("SHARED/userInvitation", "invite");
-   requireJs.addScripts("invite.build('user', '$restUrl', '$placeholder');");
+   requireJs.addScripts("invite.build('user', '$restUrl', \"$placeholder\");");
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: We should instead escape single quotes and double quotes IMO.

Comment (boubaker) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: Done

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl [hunk 2]
```diff
@@ -4,6 +4,9 @@ String inviteAction = uicomponent.event("Invite") + "; return false;";
 String restUrl = uicomponent.getRestURL();
 String value = uicomponent.getValue();
 String placeholder = _ctx.appRes("UIUserInvitation.input.placeholder");
+if(placeholder != null) {
+	placeholder = placeholder.replace("'", "\\'").replace("&#39;", "\\&#39;")
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: A little tip : in groovy you can use the safe navigation operator (http://docs.groovy-lang.org/latest/html/documentation/index.html#_safe_navigation_operator). So you can replace your block with :

`placeholder = placeholder?.replace("'", "\\'").replace("&#39;", "\\&#39;")`

Comment (boubaker) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: Done

**Comment by boubaker** [comment]
Merged

**Pull request closed without merging**